raiden/tests/utils/network.py

```diff
@@ -472,7 +473,7 @@ def wait_for_usable_channel(
     is reachable.
     """
     waiting.wait_for_newchannel(
-        app0.raiden, registry_address, token_address, app1.raiden.address, app0.raiden.address, retry_timeout
+        app0.raiden, registry_address, token_address, app0.raiden.address, app1.raiden.address, retry_timeout
```

Comment:
Why this was needed?

Reply:
It was in the wrong direction, the creator address should be the first and the partner should be the second one, and for some reason it wasn't, app0 is who creates the channel so, also this was needed because it was failing on the mapping.

Reply:
Ok, thats because i changed the `views.get_channelstate_for ` method